src/components/TextInput/TextInput.test.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
Let's pull all the tests into subdirectory `__tests__`. Keep following this structure:
```
src/components/TextInput/
  __tests__/** - Test files
  __snapshots__/** - Snapshots for tests
  __stories__/** - Storybook stories
```